lib/splitclient-rb/cache/adapters/redis_adapter.rb

```diff
@@ -118,6 +118,23 @@ def random_set_elements(key, count)
           @redis.srandmember(key, count)
         end
 
+        # Queue
+        def add_to_queue(key, val, max_size)
+          @redis.del(key) if @redis.llen(key) >= max_size
```

Comment:
@maksimf my concern here is that:  
 - if this delete key when max_size is reached is executed in redis-shared mode, the events into redis LIST will be removed and the consumer (split synchronizer) will lost those deleted events.

I think that eventsQueueSize configuration param is only to in-memory mode (not redis)

Reply:
ok, thank you, I'll fix that